core/src/main/java/org/apache/accumulo/core/client/impl/DelegationToken.java

```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.accumulo.core.client.security.tokens;
+package org.apache.accumulo.core.client.impl;
```

Comment:
> What public methods would DelegationToken have that AuthenticationToken does not?

We could hypothetically add methods which inform the user on the time-validity of the token, but I believe you already know that the answer is that there are none presently.

> I will try to create an opaque DelegationToken type. It seems like that gives us more API flexibility in the future, even if we don't have any extra public methods now.

Thank you. I feel like that is a better long term fix that gives us additional flexibility.